mono/metadata/mono-perfcounters.c

```diff
@@ -258,6 +311,8 @@ typedef struct {
 typedef struct {
 	const char *name;
 	const char *help;
+	guint name_length;
```

Comment:
These new fields seems to be read from but never written to (just initialized to 0)

Reply:
I think they are used. The last struct is zero, yes, but they are used I think otherwise.